content/guides/contribute/adding-new-components.mdx

```diff
@@ -1,69 +1,21 @@
 ---
 title: Adding new components
-description: Guidance to help you get your components added to Primer. The primary audience for this guidance is GitHub staff.
+description: Guidance to help build Primer Recipes. The primary audience for this guidance is GitHub staff.
 ---
 
-## On-ramp from product to Primer
+## New Components start as Primer Recipes
 
-The process starts with a new component being introduced as a custom React component in a feature team's app and becoming a Primer component once it matures and gets more usage.
+**Open Source contributors**: Please [open a discussion in Primer React](https://github.com/primer/react/discussions/new?category=ideas) if you'd like to provide feedback on existing components or suggest component additions for the library.
```

Comment:
Why Primer React instead of primer/design?

Reply:
This is a good question, particularly since "component additions" would have to go through design review first anyway. @tallys do you know why we thought `primer/react` was the better fit?